test/bench/bench.cpp

```diff
@@ -58,7 +58,7 @@ std::vector<BenchmarkCase::Input> load_inputs(const StateTransitionTest& state_t
 BenchmarkCase load_benchmark(const fs::path& path, const std::string& name_prefix)
 {
     std::ifstream f{path};
-    auto state_test = std::move(evmone::test::load_state_tests(f).at(0));
+    auto state_test = std::move(load_state_tests(f).at(0));
```

Comment:
Unrelated change. I would move it to separated commit.

Reply:
This is almost like a typo, we are already in the `evmone::test` namespace.